resource-watch/resource-watch
# Topics to dashboad (routes + menu label)

## Overview
This PR adds "dashboards" label to main menu instead of "topic" and updates routes from /topics to /dashboards
Topics page still available by route /topics.
All changes with dashboard content have done in topics components. (layout/topics and layout/app/topics-detail). We are planning to rename them in future PR.

## Testing instructions
From main page. Click "dashboards" label in main menu or go to /dashboards route. You should see the content from previous "topics" page.
Click on some topic on this page or follow the link http://localhost:9000/dashboards/climate

## [Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/169732634)
---

## Checklist before submitting
